contracts/base/LoansBase.sol

```diff
@@ -595,6 +592,24 @@ contract LoansBase is LoansInterface, Base {
         return loanAmount;
     }
 
+    /**
+        @notice Returns the amount of interest owed for a given loan and loan amount
+        @param loanID The id of the loan to get the owed interest
+        @param amountBorrow The principal of the loan to take out
+     */
+     function _getInterestOwed(uint256 loanID, uint256 amountBorrow)
+        internal
+        view
+        returns (uint256)
+    {
+        uint256 interestOwed = amountBorrow
+            .mul(loans[loanID].loanTerms.interestRate)
+            .mul(loans[loanID].loanTerms.duration)
+            .div(TEN_THOUSAND)
+            .div(SECONDS_PER_YEAR_4DP);
+        return interestOwed;
```

Comment:
lets just return the value instead of creating a variable

Reply:
Done